sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -48,8 +98,33 @@ public enum ServiceVersion
 }
 namespace Azure.Security.KeyVault.Administration.Models
 {
+    public partial class FullBackupDetails
```

Comment:
A lot of these properties map to existing members of Operation<T>. Should we hide these classes completely?

Reply:
If we did, did you have any thoughts on what we would return from `Operation.Value`?

Reply:
Seems that there is nothing essential to return. We can do `Operation<Response>`

Reply:
The `AzureStorageBlobContainerUri` might be essential if someone is checking on the status of a backup that they didn't initiate.  I also thought the metadata about the operation like start/end times could be interesting to admins.

Reply:
Yeah, AzureStorageBlobContainerUri might be the only thing. 

>I also thought the metadata about the operation like start/end times could be interesting to admins.

We can expose it on OperationClass directly as it's something that's accessible before the operation is completed.

Reply:
> We can expose it on OperationClass directly as it's something that's accessible before the operation is completed.

I like that! It also takes care of hiding `KeyVaultServiceError`

Reply:
`Id` would still be the `JobId`. But we can now just expose it as `Id` publicly.